tests/Bedrock.Framework.Tests/ConsumableArrayBufferWriterTests.cs

```diff
@@ -349,62 +343,45 @@ public void GetMemoryWithSizeHint_InitSizeCtor(int sizeHint)
         public void GetMemoryAndSpan()
```

Comment:
Why did this test change so much?

Reply:
Since we're using the ArrayPool there's no guarantee the buffer will be clean, so I removed this check (The for loops).
For the same reason there's no guarantee that the new buffer wont happen to contain the same bytes as the old buffer. In fact it's quite likely due to the way these tests are set up, and happened by chance in one of my runs.

Assumptions about the size and capacity of the Buffer are now much weaker, so I had to change/remove them.